scripts/pkglib.sh

```diff
@@ -164,6 +164,43 @@ function checkout_repo()
     cp ../scripts/apport_hook.py $OBS_WORKDIR/$APPORT_HOOK
 }
 
+function generate_kernel_config() {
```

Comment:
Could you add a header documenting what arguments are need and what is the expected result of the function?